### DISCUSSION TITLE
Awesome-style list for InnerSource SIG

### DISCUSSION THREAD

**Discussion by claredillon** [finos/InnerSource#30] [Q&A]
An idea for one of the next outputs from the InnerSource SIG is to put together an Awesome-style list of resources. Before we begin to formally construct something - we would love for the community to add in anything they might find useful, so that we can have some content to start with.

Also, it will help us decide the initial categories for the list.

**Comment by claredillon** [comment]
As examples, here’s the Awesome List for macOS applications: https://github.com/iCHAIT/awesome-macOS
And it references, via badge, the Awesome List of Awesome Lists (very meta): https://github.com/sindresorhus/awesome

**Reply by godds** [reply]
I believe, using the badge to reference the Awesome List of Awesome Lists is "standard".

**Reply by godds** [reply]
Another example: https://github.com/ColinEberhardt/awesome-public-streaming-datasets
(to highlight that many useful ones aren't necessarily as long as previous examples)

**Comment by godds** [comment]
**Useful Reading**
[An introduction to InnerSource (GitHub)](https://resources.github.com/whitepapers/introduction-to-innersource/)
[What is InnerSource (GitLab)](https://about.gitlab.com/topics/version-control/what-is-innersource/)
[Getting Started with InnerSource](https://innersourcecommons.org/documents/books/GettingStartedWithInnerSource.pdf) by Andy Oram
[Understanding the InnerSource Checklist](https://innersourcecommons.org/documents/books/InnerSourceChecklist.pdf) by Silona Bonewald
[Managing InnerSource Projects](https://innersourcecommons.gitbook.io/managing-inner-source-projects/) by Daniel Izquierdo & José Manrique López
[Adopting InnerSource](https://innersourcecommons.org/documents/books/AdoptingInnerSource.pdf) by Danese Cooper and Klaas-Jan Stol
[InnerSource Patterns](https://patterns.innersourcecommons.org/)

**Communities**
[InnerSource Commons](https://innersourcecommons.org/)
[FINOS InnerSource Special Interest Group](https://github.com/finos/InnerSource)

**Tools**
Our maturity matrix ☺️
[SAP Project Portal for InnerSource](https://github.com/SAP/project-portal-for-innersource)

**Reply by claredillon** [reply]
**Training**
[InnerSource Commons Learning Path ](https://innersourcecommons.org/learn/learning-path/)
